### DISCUSSION TITLE
How can I specify decider and reuse option with processing fossology on release action step

### DISCUSSION THREAD

**Discussion by piapiamilapia** [eclipse-sw360/sw360#2466] [Q&A]
In a component , I will create new release, I want to process fossology can reuse previouse release decisions that was cleared by clear expert . how can I do it? I don't find any options or configurations on sw360 UI or config file.

**Comment by GMishx** [comment]
@piapiamilapia , In the FOSSology's scan endpoint, there is an option to send `reuse` information as per following model where you can declare the option to reuse clearing from old upload:

https://github.com/fossology/fossology/blob/f157388372376e7c496e3795314ef42551b3e912/src/www/ui/api/documentation/openapi.yaml#L5370-L5393

However, SW360 currently does not have an option to define the parameters for the same and thus does not schedule it.
